src/components/CommentCard.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+function CommentCard({ comment_id, body, author, votes, created_at }) {
+  return (
+    <div>

```

Comment:
bit of a loose div here? given the structure you've gone with I can't think of anything particularly more semantic but if your comments were in a list then this could be a list item